sources/api/migration/migrations/v1.15.0/oci-defaults-resource-setting/src/main.rs

```diff
@@ -0,0 +1,43 @@
+use migration_helpers::common_migrations::{AddPrefixesMigration, NoOpMigration};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added new resource limit settings for configuring the default OCI runtime spec,
+/// `settings.oci-defaults.resource-limits`, which will initially contain only
+/// `settings.oci-defaults.resource-limits.max-open-files`. We have to remove other
+/// settings on downgrade.
```

Comment:
Should update this comment. Can just generically describe that we've added additional resource limits.